ui/components/graphql/subscriptions/ConfigurationSubscription.js

```diff
@@ -47,6 +47,7 @@ const configurationSubscription = graphql`
           id
           name
           filter_file
```

Comment:
Can we remove `filter_file` from the query?

Reply:
No, we don't need it since `filter_file` is available only in case of download otherwise it is empty. And there is a separate endpoint to download, so we can remove this.

Reply:
Be sure to run `npm run relay` and `make graphql-build` after running. And removing references of `filter_file` from UI components.